packages/muze-utils/src/common-utils.js

```diff
@@ -1548,18 +1547,10 @@ const getValueParser = config => (val) => {
 };
 
 const retrieveNearestGroupByReducers = (dataModel, ...measureFieldNames) => {
-    let nearestReducers = {};
-    let next = dataModel;
-    do {
-        const derivations = next.getDerivations();
-        if (derivations) {
-            const groupDerivation = derivations.reverse().find(derivation => derivation.op === DM_OPERATION_GROUP);
-            if (groupDerivation) {
-                nearestReducers = groupDerivation.criteria || {};
-                break;
-            }
-        }
-    } while (next = next.getParent());
+    const derivations = [...dataModel.getDerivations().reverse(), ...dataModel.getAncestorDerivations().reverse()];
+
+    const nearestReducers = defaultValue(
+        getObjProp(derivations.find(derv => derv.op === DM_OPERATION_GROUP), 'criteria'), {});
```

Comment:
this `DM_OPERATION_GROUP` constant should be exposed from datamodel. If name of this operation changes in datamodel, this condition will fail.
Expose operations from datamodel

Reply:
Done